raiden-ts/src/transport/epics/messages.ts

```diff
@@ -140,6 +147,10 @@ export function matrixMessageGlobalSendEpic(
             '',
           );
         }),
+        retryWaitWhile(
+          exponentialBackoff(config.pollingInterval, config.httpTimeout),
+          (err, count) => err?.httpStatus !== 429 || err?.httpStatus !== 500 || count > 3,
```

Comment:
```suggestion
          (err, count) => ![429, 500].includes(err?.httpStatus) || count > 3,
```
Or `!==` + `||` on 2 different constants is always true